### PR TITLE
Add n19953 notes

### PR DESCRIPTION
**Pull request by jnewbery**

### CODE REVIEW

**Review by jonatack — COMMENTED** [review]
Looks great! Just a couple fixups and looks good to go. Excellent questions!

File: _posts/2020-10-14-#19953.md [hunk 1]
```diff
@@ -19,12 +19,84 @@ _[Implement Tapscript script validation rules (BIP
 
 _Notes and questions to follow soon!_
```
[the remark this answers is not in the document]
Comment (jnewbery) [reply _posts/2020-10-14-#19953.md:20]: done

File: _posts/2020-10-14-#19953.md [hunk 2]
```diff
@@ -19,12 +19,84 @@ _[Implement Tapscript script validation rules (BIP
...
+
+- Tapscript has almost identical script semantics to [those defined in BIP
+  141](https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki#New_script_semantics)
+  for P2WSH spends, which themselves are almost identical to the semantics for
+  pre-segwit script.
+
+- The main differences between tapscript and P2WSH semantics are:
+
+  - BIP 340 schnorr signature validation is used for all `OP_CHECKSIG` and
+    `OP_CHECKSIGVERIFY` evaluation , with a new [signature hash
```
[the remark this answers is not in the document]
Comment (jnewbery) [reply _posts/2020-10-14-#19953.md:43]: removed the space.

changed the wording around signature hash a bit to hopefully make it clearer.

File: _posts/2020-10-14-#19953.md [hunk 3]
```diff
@@ -19,12 +19,84 @@ _[Implement Tapscript script validation rules (BIP
...
+    algorithm](https://github.com/bitcoin/bips/blob/master/bip-0342.mediawiki#signature-validation).
+  - `OP_CHECKMULTISIG` and `OP_CHECKMULTISIGVERIFY` are disabled, with their
+    functionality replaced by `OP_CHECKSIGADD`.
+  - Certain opcodes are redefined to mean `OP_SUCCESS` and cause validation to succeed immediately.
+  - `MINIMALIF` becomes a consensus rule.
```
[the remark this answers is not in the document]
Comment (jnewbery) [reply _posts/2020-10-14-#19953.md:48]: I'll leave this out to not give the answer away.

**Comment by jnewbery** [comment]
Thanks for the review @jonatack

**Pull request merged**